src/ansys/acp/core/extras/example_helpers.py

```diff
@@ -150,6 +152,12 @@ class ExampleKeys(Enum):
     ExampleKeys.CUT_OFF_GEOMETRY_SOLID_MODEL: _ExampleLocation(
         directory="geometries", filename="cut_off_geometry_solid_model.stp"
     ),
+    ExampleKeys.GS_MINIMAL_PLATE_NO_MATML: _ExampleLocation(
+        directory="getting_started", filename="minimal_plate_no_matml.acph5"
+    ),
+    ExampleKeys.GS_MINIMAL_PLATE: _ExampleLocation(
```

Comment:
```suggestion
    ExampleKeys.CDB_MINIMAL_PLATE: _ExampleLocation(
```

What does `GS` stand for?

Reply:
FYI: acph5 includes the input mesh (loaded from a CDB or H5) and the layup definition. A CDB contains the mesh, named selections and materials only, so no plies.

Reply:
GS is for Getting Started, I added it such that it's more explicit what the key refers to in the python code. It would be fine to remove it though.